package.json

```diff
@@ -49,6 +49,7 @@
     "@google-cloud/projectify": "^0.3.2",
     "@google-cloud/promisify": "^0.4.0",
     "@types/duplexify": "^3.6.0",
+    "@types/long": "^4.0.0",
```

Comment:
you don't need this because it's shipped with `protobufjs` :)

Reply:
Yeah, but you shouldn't depend on it existing transitively. It won't work with `yarn pnp` for instance

Reply:
ah, cool - did not know that!